kinto/tests/test_views_collections_cache.py

```diff
@@ -0,0 +1,81 @@
+from .support import (BaseWebTest, unittest, MINIMALIST_RECORD)
+
+
+class ForcedCacheExpiresTest(BaseWebTest, unittest.TestCase):
+    def setUp(self):
+        super(ForcedCacheExpiresTest, self).setUp()
+        r = self.app.post_json('/buckets/default/collections/cached/records',
+                               MINIMALIST_RECORD,
+                               headers=self.headers)
+        self.record = r.json['data']
+
+    def get_app_settings(self, extra=None):
+        settings = super(ForcedCacheExpiresTest, self).get_app_settings(extra)
+        settings['cliquet.record_cache_expires_seconds'] = 3600
+        return settings
+
+    def test_expire_and_cache_control_are_set(self):
+        url = '/buckets/default/collections/cached/records'
+        r = self.app.get(url,
+                         headers=self.headers)
+        self.assertIn('Expires', r.headers)
+        self.assertIn('Cache-Control', r.headers)
+        r = self.app.get(url + '/%s' % self.record['id'],
+                         headers=self.headers)
+        self.assertIn('Expires', r.headers)
+        self.assertIn('Cache-Control', r.headers)
+
+
+class CollectionExpiresTest(BaseWebTest, unittest.TestCase):
+    def setUp(self):
+        super(CollectionExpiresTest, self).setUp()
+        self.collection_url = '/buckets/default/collections/cached'
+        self.app.put_json(self.collection_url,
+                          {'data': {'cache_expires': 3600}},
+                          headers=self.headers)
+
+        self.records_url = self.collection_url + '/records'
+
+        resp = self.app.post_json(self.records_url,
+                                  MINIMALIST_RECORD,
+                                  headers=self.headers)
+        self.record = resp.json['data']
+        self.record_url = self.records_url + '/' + self.record['id']
+
+    def test_cache_expires_must_be_an_integer(self):
```

Comment:
should this go in another test case? The setup isn't necessary here if I read correctly.